rapidtide/correlate.py

```diff
@@ -710,7 +732,7 @@ def aliasedcorrelate(
     lowresstarttime=0.0,
     padtime=30.0,
 ):
-    """
+    """Perform an aliased correlation.
```

Comment:
The function `aliasedcorrelate` doesn't seem to be used anywhere except for a test (`test_aliasedcorrelate.py`).

Reply:
I guess it's function is fully taken over by the aliasedcorrelator class.  Let's keep it around until I fix the test to use the class.